src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -396,17 +395,17 @@ const WorkspaceDashboard = _.flow(
       ]),
       h(RightBoxSection, {
         title: 'Tags',
-        info: span({ onClick: e => e.stopPropagation() }, [
-          h(InfoBox, { style: { marginLeft: '0.25rem' } }, [
-            `${getAppName()} is not intended to host personally identifiable information. Do not use any patient identifier including name,
-            social security number, or medical record number.`
-          ]),
-          (busy || !tagsList) && tagsPanelOpen && spinner({ size: '1.5ch', style: { marginLeft: '0.5rem' } })
+        info: span({}, [
+          (busy || !tagsList) && tagsPanelOpen && spinner({ size: '1ch', style: { marginLeft: '0.5rem' } })
         ]),
         initialOpenState: tagsPanelOpen,
         onClick: () => setTagsPanelOpen(!tagsPanelOpen)
       }, [
         div({ style: { margin: '0.5rem' } }, [
+          div({ style: { marginBottom: '0.5rem' } }, [
+            `${getAppName()} is not intended to host personally identifiable information. Do not use
```

Comment:
Was this the suggestion from UX? I thought that perhaps we could put the info button near the search field, or at least make the text smaller, but defer to UX.

Reply:
eb625345380f2d8eecd01070e75707292994d3e4

Reply:
I'm out this afternoon anyway… so I won't get back to this until tomorrow unfortunately. You can check in with Ean Moody, who is the new UX person working with the Workspaces team.

Reply:
He was fine with it but we ultimately landed on this abbreviated version:

<img width="453" alt="Screen Shot 2022-04-21 at 1 32 26 PM" src="https://user-images.githubusercontent.com/7257391/164517815-c30c3315-a1f7-4c62-98d7-ed029f6873bf.png">